src/main/java/com/squareup/pagerduty/incidents/PagerDuty.java

```diff
@@ -27,18 +27,18 @@
 
   /** Create a new instance using the specified API key. */
   public static PagerDuty create(String apiKey) {
-    RestAdapter restAdapter = new RestAdapter.Builder() //
-        .setEndpoint(Endpoints.newFixedEndpoint(HOST)) //
+    Retrofit retrofit = new Retrofit.Builder() //
+        .baseUrl(HOST) //
```

Comment:
With v1, Gson was implicitly used. In v2 there is no default converter so the lack of installing a converter factory here will cause it to fail at runtime.